drivers/cc2420/include/cc2420_params.h

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
hm, this is the place we put it for all (network) devices so far... If we move it, we should IMHO move it for all of them, so separate issue?!